gradle.properties

```diff
@@ -43,7 +43,5 @@ POM_DEVELOPER_URL=https://github.com/slackhq
 POM_INCEPTION_YEAR=2022
 POM_PACKAGING=jar
 SONATYPE_STAGING_PROFILE=com.slack
```

Comment:
Both POM_PACKAGING and SONATYPE_STAGING_PROFILE are not used anymore

Reply:
hmm, does it just view whatever is visible to the account being used?

Reply:
e96c2e4

Reply:
It retrieves the staging profiles in your account and finds the one that equals or is a prefix of the group id